### ISSUE TITLE
Update Agreement Addition

### ISSUE THREAD

**Issue by 06chaynes** [br0xen/ConnectWise_PHP#2]
Let me start by saying thank you for this great class! It has saved me quite some time. This isn't so much as an issue as it is a request for information. Could you show me an example of updating an agreement addition using your class? I am trying to update the 'BillCustomer' field but I am having some issues in formatting the request properly.

**Comment by br0xen** [comment]
Hey 06chaynes, ConnectWise has updated their API since I last updated this repo. I just pushed a new commit that has some changes that I've implemented. If you're still using the old ConnectWise API, then I'm not sure what the correct way would be to do what you want to do (because I don't think CW even has their old API docs online, at least not that I found).

If your CW instance has been updated, though, you might give the 2.0 files a shot that are in the new commit.
If you're using the CodeIgniter library: /CI_Library/application/libraries/connectwise2.php
If you're using the regular PHP library: /php_class/connectwise2.php

If you're using the regular PHP library, beware, I haven't tested that one nearly as much as the CI library.

Let me know how it goes or if you continue to have issues. I'll leave this issue open until I hear back (or a significant amount of time passes. ;-)

Thanks!
